Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/StringUtil.java

```diff
@@ -45,18 +45,16 @@
      * @param tokens
      * @return String
      */
-    public static String extractStringFromTokens(String tokenString, String tokens) {
+    public static String extractStringFromTokens(final String tokenString, final String tokens) {
         String resultString = "";
-        if (tokens != null && !tokens.isEmpty()) {
-            StringTokenizer tk = new StringTokenizer(tokenString, tokens);
-            StringBuffer outString = new StringBuffer();
-            while (tk.hasMoreTokens()) {
-                outString.append(tk.nextToken());
-            }
-            resultString = outString.toString();
-        } else {
-            resultString = tokenString;
+
+        final StringTokenizer tk = new StringTokenizer(tokenString, tokens);
+        final StringBuffer outString = new StringBuffer();
+        while (tk.hasMoreTokens()) {
```

Comment:
Are we guaranteed that  tokenString and tokens will always be not null when the extractStringFromTokens method is called?  If not, then  while (tk.hasMoreTokens()) wil throuw a NPE.
Reference: https://docs.oracle.com/javase/7/docs/api/java/util/StringTokenizer.html#StringTokenizer(java.lang.String,%20java.lang.String)